app/controllers/ProductController.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace controllers;
+
+use src\Classes\Controller;
+
+class ProductController extends Controller
+{
+    public function index(string $path): void
+    {
+        $modelPath = "models\\" . $path . "Model";
```

Comment:
weird system design, but interesting. you can move this logic of getting the model to the Request class, by implementing everything for initiating models there. And, instead of using `string $path` args, pass initiated Request instance to the controller from the Router

Reply:
changed